apps/test/unit/templates/sectionProgress/ProgressTableLevelBubbleTest.jsx

```diff
@@ -80,41 +82,73 @@ describe('ProgressTableLevelBubble', () => {
     );
     expect(wrapper.find(unitTestExports.LinkWrapper)).to.have.lengthOf(0);
   });
 
+  it('renders default bubble with circle shape', () => {
+    const wrapper = mount(<ProgressTableLevelBubble {...defaultProps} />);
+    expect(wrapper.find(unitTestExports.BasicBubble).props().shape).to.equal(
+      BubbleShape.circle
+    );
+  });
+
+  it('renders concept bubble with diamond shape', () => {
+    const wrapper = mount(
+      <ProgressTableLevelBubble {...defaultProps} isConcept={true} />
+    );
+    expect(wrapper.find(unitTestExports.BasicBubble).props().shape).to.equal(
+      BubbleShape.diamond
+    );
+  });
+
+  it('renders unplugged bubble with pill shape', () => {
+    const wrapper = mount(
+      <ProgressTableLevelBubble {...defaultProps} isUnplugged={true} />
+    );
+    expect(wrapper.find(unitTestExports.BasicBubble).props().shape).to.equal(
+      BubbleShape.pill
+    );
+  });
+
   it('shows correct text in unplugged bubble', () => {
     const wrapper = mount(
-      <ProgressTableLevelBubble {...defaultProps} unplugged={true} />
+      <ProgressTableLevelBubble {...defaultProps} isUnplugged={true} />
     );
-    expect(wrapper.find(unitTestExports.UnpluggedBubble)).to.have.lengthOf(1);
-    expect(wrapper.find(unitTestExports.Content).text()).to.equal(
+    expect(wrapper.find(unitTestExports.BasicBubble).text()).to.equal(
       i18n.unpluggedActivity()
     );
   });
 
   it('shows title in normal bubble', () => {
     const wrapper = mount(<ProgressTableLevelBubble {...defaultProps} />);
-    expect(wrapper.find(unitTestExports.LargeCircle)).to.have.lengthOf(1);
-    expect(wrapper.find(unitTestExports.Content).text()).to.equal(TITLE);
+    expect(wrapper.find(unitTestExports.BasicBubble).text()).to.equal(TITLE);
   });
 
   it('shows title in concept bubble', () => {
     const wrapper = mount(
       <ProgressTableLevelBubble {...defaultProps} concept={true} />
     );
-    expect(wrapper.find(unitTestExports.LargeDiamond)).to.have.lengthOf(1);
-    expect(wrapper.find(unitTestExports.Content).text()).to.equal(TITLE);
+    expect(wrapper.find(unitTestExports.BasicBubble).text()).to.equal(TITLE);
+  });
+
+  it('shows title in letter bubble', () => {
+    const wrapper = mount(
+      <ProgressTableLevelBubble
+        {...defaultProps}
+        bubbleSize={BubbleSize.letter}
+      />
+    );
+    expect(wrapper.find(unitTestExports.BasicBubble).text()).to.equal(TITLE);
   });
 
-  it('shows title in small bubble', () => {
+  it('does not show title in dot bubble', () => {
     const wrapper = mount(
-      <ProgressTableLevelBubble {...defaultProps} smallBubble={true} />
+      <ProgressTableLevelBubble {...defaultProps} bubbleSize={BubbleSize.dot} />
     );
-    expect(wrapper.find(unitTestExports.SmallCircle)).to.have.lengthOf(1);
-    expect(wrapper.find(unitTestExports.Content).text()).to.equal(TITLE);
+    console.log('text', wrapper.find(unitTestExports.BasicBubble).text());
```

Comment:
Rogue log 👀

Reply:
oof. okay, gotta push again so i'll fix the above nit as well.